packages/grid/x-data-grid-generator/src/services/basic-data-service.ts

```diff
@@ -0,0 +1,106 @@
+import { GridColDef, GridRowId } from '@mui/x-data-grid';
+
+export const currencyPairs = [
+  'USDGBP',
+  'USDEUR',
+  'GBPEUR',
+  'JPYUSD',
+  'MXNUSD',
+  'BTCUSD',
+  'USDCAD',
+  'EURJPY',
+  'EURUSD',
+  'EURCHF',
+  'USDCHF',
+  'EURGBP',
+  'GBPUSD',
+  'AUDCAD',
+  'NZDUSD',
+  'GBPCHF',
+  'AUDUSD',
+  'GBPJPY',
+  'USDJPY',
+  'CHFJPY',
+  'EURCAD',
+  'AUDJPY',
+  'EURAUD',
+  'AUDNZD',
+  'CADEUR',
+  'CHFMXN',
+  'ETHUSD',
+  'BCHUSD',
+  'ETHBTC',
+  'XRPUSD',
+  'XRPBTC',
+  'USDLTC',
+  'USDXRP',
+  'USDDSH',
+  'USDBCH',
+  'JPYEUR',
+  'JPYUSD',
+  'JPYGBP',
+  'JPYCAD',
+  'JPYCHF',
+  'JPYAUD',
+  'JPYNZD',
+  'JPYSGD',
+  'JPYNOK',
+  'JPYRUB',
+  'JPYSEK',
+  'JPYTRY',
+  'JPYZAR',
+  'JPYHKD',
+  'JPYCNH',
+  'JPYDKK',
+  'JPYMXN',
+  'JPYPLN',
+  'JPYXAG',
+  'JPYXAU',
+  'JPYBTC',
+  'JPYETH',
+  'JPYLTC',
+  'JPYXRP',
+  'JPYDSH',
+  'JPYBCH',
+  'GBPEUR',
+  'GBPRUB',
+  'GBPTRY',
+];
+
+interface GridBasicRowModel {
+  id: GridRowId;
+  currencyPair: string;
+  [price: string]: number | string;
+}
+
+export interface GridBasicData {
+  columns: GridColDef[];
+  rows: GridBasicRowModel[];
+}
+
+export const getBasicGridData = (rowLength: number, colLength: number): GridBasicData => {
```

Comment:
Nice, I guess this was part of the storybook package. I wanter if we can remove it altogether and just use either the `Commodity` or `Employee` data sets. If there are no assertions in the tests related to the value of the cells then replacing the data set should be more or less easy, what do you think?

Reply:
Yep, it's the old `useData` and `getData`.

For the tests, it can be useful to have those dummy data tests without any intelligence behind them.
For instance in the pagination, it makes very clear what you are testing.

I would personally be in favor of an even dumber data set.
Something like:

```
{ id: 0, name: `name-0` },
{ id: 1, name: `name-1\ },
...
```

And to use it everywhere we have some "Nike" / "Puma" dataset.
To reduce as much as possible the complexity of understanding the dataset and focusing on the actual behavior we are testing.